docs/guides/run-elasticsearch.rst

```diff
@@ -0,0 +1,79 @@
+==========================================
+Enabling Elasticsearch on the local server
+==========================================
+
+Read the Docs has been using Elasticsearch which is a platform for distributed search and analysis of data in real time. To enable the search feature on your local installation, you need to install the elasticsearch locally and run the Elastic server. 
+
+Installation has been mainly divided into following steps.
+
+1. Installing Java
+------------------
+
+Elasticsearch requires JAVA 8 or later. Use .. _Oracle official documentation:http://www.oracle.com/technetwork/java/javase/downloads/index.html or opensource distribution like .. _OpenJDK:http://openjdk.java.net/install/
+
+After installing java, verify the installation by,::
+
+	(READTHEDOCS)$ java -version
+
+The result should be something like this::
+
+	openjdk version "1.8.0_151"
+	OpenJDK Runtime Environment (build 1.8.0_151-8u151-b12-0ubuntu0.16.04.2-b12)
+	OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)
+
+
+2. Downloading and installing Elasticsearch
+-------------------------------------------
+
+Elasticsearch can be downloaded directly from elastic.co in zip, tar.gz, deb, or rpm packages. For Ubuntu, it's best to use the deb (Debian) package which will install everything you need to run Elasticsearch.
+
+RTD currently uses elasticsearch 1.x which can be easily downloaded and installed from the official website (http://elastic.co).
+
+Download it in a directory parallel to where the readthedocs.org project has been stored.::
```

Comment:
This is strictly necessary? What about the people that install it with a package manager? (also I think this step is visible on the elastic search official guide)

Reply:
After some research, I found out that there are various ways to start the elastic server and the steps I have written works as well but requires a bit more work than the installation with package manager. 
I guess i'm gonna re-write the guide for installing and setting up elastic server via package manager.  :)